### ISSUE TITLE
Results between training previews and after merge are really different

### ISSUE THREAD

**Issue by scarbain** [tripplyons/oft#5]
Hi!

I'm training using the default values you set.
After a few thousands of steps, the training previews are starting to be overfitted but if I take the corresponding attention processor and merge it with my base model, the inference images in automatic1111 are really undertrained. I can see that the base model have changed (comparing with same seed) and it's "starting" to converge to my concept but it's still WAY undertrained, even after 10000 steps at LR1e-4.

Any idea to what this is due ? Is it because of a configuration in my automatic1111 or the conversion script ?
 
How many steps have you used in your tests ? I've made some tests with 5 and with 200 images with different concepts, same problem.

Thanks

**Comment by tripplyons** [comment]
Hi,

I have not encountered this issue yet. So far, I have had similar images during validation and using the model after merging.

I have tested with 5 images of myself and it seems to start overfitting at 400 steps (80 epochs).

It sounds like you are following the right steps, so I might need to see more details. Do you have a Discord account or some other way of messaging for us to discuss further?

**Comment by scarbain** [comment]
Sure! Here is my discord : mamad851
